static/js/src/advantage/contracts-api.js

```diff
@@ -7,6 +7,25 @@ export async function getPurchase(purchaseID) {
   return data;
 }
 
+export async function getPurchaseAccount(email, paymentMethodID) {
+  let response = await fetch(`/advantage/purchase-account`, {
+    method: "POST",
+    cache: "no-store",
+    credentials: "include",
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      email: email,
```

Comment:
I said we didn't take fields other than this (and that is still true), but there were discussions today about accepting the customer name in this endpoint as well for guest purchases. We're waiting for feedback on whether this is really needed. Just something to have in mind as a probable future modification.